rp2040-hal/src/uart/writer.rs

```diff
@@ -164,6 +177,11 @@ impl<D: UartDevice, P: ValidUartPinout<D>> Writer<D, P> {
     pub fn disable_tx_interrupt(&mut self) {
         disable_tx_interrupt(&self.device)
     }
+
+    /// Initiates a break
+    pub fn send_break(&mut self) -> Break<'_, D> {
```

Comment:
A doc comment showing how to use this would be nice.
Otherwise, just seeing this function in the docs, it's not obvious how to control the length of the break.